packages/core/src/Stores/client-store.js

```diff
@@ -340,6 +344,17 @@ export default class ClientStore extends BaseStore {
         return false;
     }
 
+    @computed({ keepAlive: true })
+    get is_eu_country() {
+        let clients_country = Cookies.get('clients_country');
```

Comment:
for deriv-app I don't think we need a cookie for clients_country we can directly use the `this.website_status.clients_country` from the store